src/rmodels.c

```diff
@@ -1383,7 +1383,7 @@ void DrawMesh(Mesh mesh, Material material, Matrix transform)
     }
 
     // Get a copy of current matrices to work with,
-    // just in case stereo render is required and we need to modify them
+    // just in case stereo render is required, and we need to modify them
```

Comment:
The "and" here may be stronger than the conjunction it is with the comma.  I will not die on this though.